x/nexus/keeper/keeper_test.go

```diff
@@ -41,6 +41,8 @@ func addressValidator() types.Router {
 			switch chain {
 			case axelarnet.Axelarnet.Name:
 				prefix = "axelar"
+			case "terra-2":
```

Comment:
why is it only terra?

Reply:
Terra defaults to itself for the prefix name but I can make it explicit